Tangem/Modules/SingleTokenBase/SingleTokenBaseViewModel.swift

```diff
@@ -249,7 +250,8 @@ extension SingleTokenBaseViewModel {
     }
 
     private func updatePendingTransactionView() {
-        guard transactionHistoryState == .notSupported else {
+        // Only if the transaction history isn't supported
+        guard !walletModel.isSupportedTransactionHistory else {
```

Comment:
мммм, мое любимое, что мы не обсудили)))

```suggestion
        if walletModel.isSupportedTransactionHistory {
```

Reply:
Надо обсудить, тут просто как раз в начале метода хочется `guard `